pkg/webhook/stats_reporter.go

```diff
@@ -65,6 +67,19 @@ func (r *reporter) ReportAdmissionRequest(response admissionReqRes, d time.Durat
 	return r.report(ctx, responseTimeInSecM.M(d.Seconds()))
 }
 
+// Captures req count metric, recording the count and the duration
+func (r *reporter) ReportMutationRequest(response mutationResponse, d time.Duration) error {
```

Comment:
can we combine `ReportMutationRequest` and `ReportAdmissionRequest` into `ReportRequest` and pass in applicable status key?

Reply:
Are you suggesting reporting both under the same metric?

We probably want to separate mutation from validation, as they will likely have very different performance profiles.

Also, because mutation is called serially, it has different operating constraints.

Reply:
Added #976 to move the discussion there.
The change is originally made in #881, this PR just depends on it.